src/photos/targets/services/onPhotoUpload.js

```diff
@@ -137,27 +142,40 @@ const recomputeParameters = async setting => {
   }
   log('info', `Compute clustering parameters on ${files.length} photos`)
 
-  const dataset = prepareDataset(files)
+  const dataset = await prepareDataset(files)
```

Comment:
The function wasn't async before?

Reply:
Oh right, this function turns async in my modifications, but it is not part of this PR. I change this.